common/data_refinery_common/models/sample_annotation.py

```diff
@@ -19,6 +19,7 @@ class Meta:
     sample = models.ForeignKey("Sample", blank=False, null=False, on_delete=models.CASCADE)
 
     # Properties
+    name = models.TextField(null=True)
```

Comment:
Do we use this field anywhere?

Reply:
Oh good catch, I had added this because I was going to record the file report as an annotation but then decided to instead just stick it directly on that model.